js/gated-content/src/components/blog/BlogListing.vue

```diff
@@ -171,6 +178,26 @@ export default {
         };
       }
 
+      if (this.categories !== null) {
```

Comment:
In case we have empty array default value:
`if (Array.isArray(this.categories) && this.categories.length)`